app/views/patients/_patient_dashboard.html.erb

```diff
@@ -10,7 +10,7 @@
         <%= f.select :last_menstrual_period_weeks, options_for_select(weeks_options, patient.last_menstrual_period_weeks ), label: 'LMP at intake', autocomplete: 'off', help: "Currently: #{patient.last_menstrual_period_display_short}" %>
       </div>
       <div class="col-sm-6 hide-label">
-        <%= f.select :last_menstrual_period_days, options_for_select(days_options, patient.last_menstrual_period_days ), label: '.', autocomplete: 'off', help: "Called on: #{patient.initial_call_date}" %>
+        <%= f.select :last_menstrual_period_days, options_for_select(days_options, patient.last_menstrual_period_days ), label: '.', autocomplete: 'off', help: "Called on: #{patient.initial_call_date.strftime("%m/%d/%Y")}" %>
```

Comment:
Should be able to use a `patient.initial_call_date.display_date` here rather than the strftime.